ref : fix-foldername-GitHub3

hnalpha323 opened the pull request:
Fix folder name to match namespace, coding style and solution file.


Fixes : Restore and build failing in Linux #12347 

## Proposed changes

- Renames the folder to conform to namespace , coding style and solution reference.

## Merge strategy

<!-- Change the following if the merge strategy should be changed:
- Squash merge (maintainer to decide merge message, PR submitter should cleanup commits/messages at PR approval).
- Rebase merge (PR submitter must change the commit message for the last commit).
- Merge commit. (PR submitter to rebase and squash before merges).
- To be decided later.
The maintainer may still request the contributor to squash and rebase, to make sure that merges and commit messages are clarified.
-->

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

mstv reviewed (commented):
This may compile but would create two folders.
I.e. all files in the folder should be renamed.

![image](https://github.com/user-attachments/assets/b2e0c3e4-3f06-4a85-8074-274d10a8e6bc)

---

And add your sign-off to `contributors.txt`, please.

hnalpha323 commented:
> This may compile but would create two folders. I.e. all files in the folder should be renamed.
> 
> ![image](https://private-user-images.githubusercontent.com/36601201/440126894-b2e0c3e4-3f06-4a85-8074-274d10a8e6bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDYyODUyMjUsIm5iZiI6MTc0NjI4NDkyNSwicGF0aCI6Ii8zNjYwMTIwMS80NDAxMjY4OTQtYjJlMGMzZTQtM2YwNi00YTg1LTgwNzQtMjc0ZDEwYThlNmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTAzVDE1MDg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmNGViMWIwNDkwNGJkZjFjNjU0ZTIyNjFlN2M3NTBlMTE5ODQ3YWYyNWJmMzdlYzY4ZTcwZDc2OTM3M2ZmZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.cVFGcgFaqLdtwc3wSNankE8Fg7FF9OetE663v947LW8)
> 
> And add your sign-off to `contributors.txt`, please.

You are absolutely right, Well this would be my first and last time using GitHub UI to make a change.  
I fixed it and added my name to contributors.txt as well

mstv reviewed (approved):
Thank you
